doc/nrf/ug_nrf5340.rst

```diff
@@ -139,7 +139,19 @@ Depending on the sample, you must program only the application core (for example
    You can use the :ref:`nrf5340_empty_app_core` sample for this purpose.
    For details, see the code in :file:`zephyr/boards/arm/nrf5340pdk_nrf5340/nrf5340_cpunet_reset.c`.
 
-Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
+To program only the application core, follow the instructions in :ref:`gs_programming_ses` and use ``nrf5340pdk_nrf5340_cpuapp`` or ``nrf5340pdk_nrf5340_cpuappns`` as build target.
+
+When programming both the application core and the network core, you can choose whether you want to build and program both images separately or combined as a :ref:`multi-image build <ug_multi_image>`.
+
+In a multi-image build, the image for the application core is the parent image, and the image for the network core is treated as a child image in a separate domain.
+For this to work, the network core image must be explicitly added as a child image to one of the application core images.
+See :ref:`ug_multi_image_defining` for details.
+
+The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
```

Comment:
Is it correct that it is added to all BLE samples? If not, when is it added?

Reply:
It is added if `CONFIG_BT_RPMSG_NRF53` is set. By default all bluetooth samples set this config for nrf53.

```
# Special handling for the nRF53 when using RPMSG HCI.
# Automatically include the hci_rpmsg sample as child image, and change
# the board to be the network core.
if (CONFIG_BT_RPMSG_NRF53)
  if ((BOARD STREQUAL nrf5340pdk_nrf5340_cpuappns) OR
      (BOARD STREQUAL nrf5340pdk_nrf5340_cpuapp))
    add_child_image(
      NAME hci_rpmsg
      SOURCE_DIR ${ZEPHYR_BASE}/samples/bluetooth/hci_rpmsg
      DOMAIN nrf5340pdk_nrf5340_cpunet)
  else()
    message(FATAL_ERROR
      "Board ${BOARD} not supported for including hci_rpmsg as child image")
  endif()
endif()
```